library/include/G4CMPConfigManager.hh

```diff
@@ -194,6 +212,14 @@ private:
   G4bool chargeCloud;    // Produce e/h pairs around position ($G4CMP_CHARGE_CLOUD) 
   G4bool recordMinE;     // Store below-minimum track energy as NIEL when killed
   G4VNIELPartition* nielPartition; // Function class to compute non-ionizing ($G4CMP_NIEL_FUNCTION)
+  // Empirical Lindhard Model Parameters
+  G4double Empiricalklow;  // Empirical Lindhard klow value
+  G4double Empiricalkhigh; // Empirical Lindhard klow value
+  G4double EmpiricalElow;  // Empirical Lindhard Elow value
+  G4double EmpiricalEhigh; // Empirical Lindhard Ehigh value
+  G4double EmpiricalkFixed; // Empirical Lindhard variable k value
+  G4bool EmpiricalEnergyDependentK; // Flag to use Empirical Lindhard with energy-dependent k
```

Comment:
I would prefer shorter variable names (look at the names on lines near where you put these).  The comments on each line are not useful, since all you're doing is repeating the variable names with spaces.  Maybe put a second line _explaining the purpose_ of the variables.